### PR TITLE
Update player_pool.hpp

### PR DESCRIPTION
**Pull request by akosd127**
Fix #792

### CODE REVIEW

**Comment by Hual** [comment]
It would be nice if you would test your code before making a PR

File: Server/Source/player_pool.hpp [hunk 1]
```diff
@@ -1655,14 +1655,16 @@ struct PlayerPool final : public IPlayerPool, public NetworkEventHandler, public
...
 		uint16_t port = player.netData_.networkID.port;
-		core.logLn(
-			LogLevel::Message,
-			"[connection] incoming connection: %s:%d id: %d",
-			addressString.data(),
-			port,
-			player.poolID);
-		playerConnectDispatcher.dispatch(&PlayerConnectEventHandler::onIncomingConnection, peer, addressString, port);
-
+		if (config.getBool("logging.log_connection_messages"))
```
Comment (Hual) [inline Server/Source/player_pool.hpp:1658]: getBool returns a pointer that needs to be dereferenced

File: Server/Source/player_pool.hpp [hunk 2]
```diff
@@ -1655,14 +1655,16 @@ struct PlayerPool final : public IPlayerPool, public NetworkEventHandler, public
...
+		{
+			core.logLn(
+				LogLevel::Message,
+				"[connection] incoming connection: %s:%d id: %d",
+				addressString.data(),
+				port,
+				player.poolID);
+			playerConnectDispatcher.dispatch(&PlayerConnectEventHandler::onIncomingConnection, peer, addressString, port);
```
Comment (Hual) [inline Server/Source/player_pool.hpp]: calling the event shouldn't be in the if statement

**Review by Hual — CHANGES_REQUESTED** [review]

File: Server/Source/player_pool.hpp (hunk 1 above)
Comment (akosd127) [reply Server/Source/player_pool.hpp:1658]: I'm sorry if I did something wrong. I just copied your code thinking it was the right way. please fix it properly and add it to the master branch. Unfortunately, English is not my native language and I don't work in IT, I'm just an avid fan

**Review by akosd127 — COMMENTED** [review]
check please

File: Server/Source/player_pool.hpp [hunk 3]
```diff
@@ -1655,14 +1655,16 @@ struct PlayerPool final : public IPlayerPool, public NetworkEventHandler, public
 		PeerAddress::AddressString addressString;
 		PeerAddress::ToString(player.netData_.networkID.address, addressString);
 		uint16_t port = player.netData_.networkID.port;
-		core.logLn(
-			LogLevel::Message,
-			"[connection] incoming connection: %s:%d id: %d",
-			addressString.data(),
-			port,
-			player.poolID);
+		if (config.getBool("logging.log_connection_messages"))
```
Comment (Hual) [inline Server/Source/player_pool.hpp:1658]: This should be *config.getBool(), same with the disconnect message; not sure how that one got in

**Review by Hual — CHANGES_REQUESTED** [review]

**Pull request closed without merging**